### PR TITLE
Updating doc and adding docker config

### PR DESCRIPTION
**Pull request by ddattee**
Doc update:
- readme update
- adding contriguting doc

Adding docker file to build dev container

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM php:5.6-apache
```
Comment (ronan-gloo) [inline Dockerfile]: php:5.6-cli-alpine should be enough, or maybe you plan to use it against a web-server ?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+ARG MOUNTPOINT=/var/www
+ARG PHPUNIT_VERSION=5
+ARG COMPOSER_BIN_DIR=/usr/local/bin
+
+RUN apt-get update -yqq \
+    && apt-get install git zlib1g-dev zip -yqq \
+    && docker-php-ext-install zip
```
Comment (ronan-gloo) [inline Dockerfile]: - Why zip ?
- We miss xml extension for feed-generator
- Also, I think that xdebug extension should be enabled in order to run code coverage

**Comment by ronan-gloo** [comment]
Thanks @ddattee. Just few notes on docker setup.

File: Dockerfile (hunk 1 above)
Comment (ddattee) [reply Dockerfile]: Yea the point is to be able to connect to the docker to run everything in the environment.
Also it allows to stay as close to most webserver configuration use by potential users.

Comment (ronan-gloo) [reply Dockerfile]: Yes, but the SDK don't run against a web-server, so it seem's useless for the library env

File: Dockerfile (hunk 2 above)
Comment (ddattee) [reply Dockerfile]: - Zip is required by composer to work fast
- OK for xml
- I plan on adding xdebug when starting to develop to set the proper configuration right

**Pull request merged**